lesson6/main.js

```diff
@@ -0,0 +1,34 @@
+
+const lists = [
+  { to: "bookmark.html", img: "1.png", alt: "画像1", text: "ブックマーク" },
+  { to: "message.html", img: "2.png", alt: "画像2", text: "メッセージ" }
+]
+
+function getListsObj() {
+  return new Promise((resolve) => {
+    setTimeout(() => {
+      resolve(lists);
+    }, 3000);
+  });
+}
+
+getListsObj().then((date) => {
+  const ul = document.getElementById('js-ul');
+  const frag = document.createDocumentFragment();
+
+  date.forEach(value => {
```

Comment:
Is it `data` instead of `date`?

Reply:
I made an embarrassing mistake🙀
Thank you for telling me.
I've made the correction.